src/github.com/cloudfoundry-incubator/switchboard/runner/monitor/cluster_monitor.go

```diff
@@ -49,7 +49,7 @@ func NewClusterMonitor(client UrlGetter, useTLSForAgent bool, backends []*domain
 
 func (c *ClusterMonitor) Monitor(stopChan <-chan interface{}) {
 	backendHealthMap := make(map[*domain.Backend]*BackendStatus)
-
+	backendHealthMap = make(map[*domain.Backend]*BackendStatus)
```

Comment:
Everything else in here makes perfect sense - this is the only line that seems unnecessary

Probably a nit, but this looks like it should be safe to delete